directives/row.py

```diff
@@ -0,0 +1,126 @@
+"""
+Directive for creating nested multirow columns with Bootstrap.
+
+Designed to be used with the point-of-interest directive.
+
+Options
+:width: sets column width. Maximum value 12.
+:column_class: Bootstrap classes can be applied. See example below.
+
+More about the Bootstrap classes https://getbootstrap.com/docs/4.4/layout/grid/#row-columns
+
+
+.. point-of-interest:: Test
+
+   .. row::
+
+     .. column::
+       :width: 8
+       :column_class: bg-warning text-center
+
+       .col-8 this is column's content.
+
+       .. row::
+
+         .. column::
+           :width: 6
+           :column_class: bg-light
+
+           .col-6
+
+         .. column::
+           :width: 6
+           :column_class: bg-secondary
+
+           .col-6
+
+     .. column::
+       :width: 4
+       :column_class: bg-success
+
+       .col-4
+
+"""
+
+from docutils.parsers.rst import Directive, directives
+from docutils import nodes
+
+import aplus_nodes
+
+
+class ColumnNode(nodes.General, nodes.Element):
+
+    def __init__(self, text):
+        super(ColumnNode, self).__init__()
+
+    @staticmethod
+    def visit_column(self, node):
+        self.body.append(self.starttag(node, 'div'))
+
+    @staticmethod
+    def depart_column(self, node=None):
+        self.body.append('</div>\n')
+
+
+class Column(Directive):
+    option_spec = {'width': directives.unchanged,
+                   'column-class': directives.unchanged,
+                   }
+
+    final_argument_whitespace = True
+    has_content = True
+
+    def run(self):
+        self.assert_has_content()
+        if 'width' in self.options:
+            # Bootstrap 3 does not support automatic widths in columns, but bootstrap 4 supports.
+            col_width = str(self.options['width']).strip()
+            if not col_width:
+                # width is empty.
+                col_width = ""
+            else:
+                # not empty
+                col_width = "-" + col_width
+        else:
+            col_width = ""
+
+        if 'column-class' in self.options:
+            column_class = str(self.options['column-class'])
+            if not column_class:
+                column_class = ""
+            else:
+                column_class = " " + column_class
+        else:
+            column_class = ""
+        column_content = '\n'.join(self.content)
+        node = ColumnNode(column_content)
+        node['classes'].append('col-sm' + col_width + column_class)
+        self.state.nested_parse(self.content, self.content_offset, node)
+        return [node]
+
+
+class Row(Directive):
+    required_arguments = 0
```

Comment:
`required_arguments = 0` is the default value so you don't need to define it again.